package Exception.defindException;

class MyException extends Exception {
    void f() {
        System.out.println("num<0,error!,注册失败!");
    }
}
